Remove _get_wrapped_func

dreiss opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* #35616 Remove (most) Python 2 support from CI scripts
* #35619 Remove _reset_warning_registry
* #35627 Remove __future__ imports from many files
* #35622 Remove caffe2/python/compatibility
* #35625 Remove structseq_slice
* #35614 Remove (most) Python 2 support from C++ code
* #35624 Remove a use of exec
* #35623 Remove a use of checkScript(str)
* **#35621 Remove _get_wrapped_func**
* #35620 Remove _compatible_subtest
* #35618 Remove _uses_true_division
* #35617 Remove (most) Python 2 support from setup.py

Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)

dreiss committed:
Remove _get_wrapped_func

Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

mrshenli reviewed (approved):
Stamp for changes in `test_c10d.py`

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)

[ghstack-poisoned]

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)

[ghstack-poisoned]

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)

[ghstack-poisoned]

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)

dreiss committed:
Update on "Remove _get_wrapped_func"


Summary:
Python 2 has reached end-of-life and is no longer supported by PyTorch.
`func.__wrapped__` can be used directly in Python 3.

Test Plan:
CI

Differential Revision: [D20842875](https://our.internmc.facebook.com/intern/diff/D20842875)
